README.md

```diff
@@ -81,6 +67,17 @@ See [building notes](/docs/build.md) for more information.
 
 At this time, Windows is not a supported platform.
 
+## Launching a Kubernetes cluster hosted on AWS
+
+Check out our [tutorial](/docs/aws.md) for launching a Kubernetes cluster hosted
+on AWS.
+
+<p align="center">
```

Comment:
I felt the same way, it's attention drawing and I love that, maybe I'll change my mind on this.  I was thinking "I want to install this! > I want to see it" but maybe having the focus be on "Why should I install this anyway?" makes sense.

I'll handle the rest of the comments today!